ui/src/App.js

```diff
@@ -3,36 +3,72 @@ import Authorize from './pages/Authorize';
 import CreateAccount from './pages/CreateAccount';
 import Loading from './pages/Loading';
 import SignIn from './pages/SignIn';
+import Reset from "./pages/Reset";
+import CreateOrRecover from './pages/CreateOrRecover';
+import Recover from './pages/Recover';
+import ViewSecretPhrase from './pages/ViewSecretPhrase';
 import { Container } from 'semantic-ui-react';
 import { appWindow } from '@tauri-apps/api/window';
 import { invoke } from '@tauri-apps/api/tauri';
-import { listen, once } from '@tauri-apps/api/event';
-import { useState, useEffect } from 'react';
+import { listen } from '@tauri-apps/api/event';
+import { useState, useEffect, useRef } from 'react';
 
 const LOADING_PAGE = 0;
 const CREATE_ACCOUNT_PAGE = 1;
 const LOGIN_PAGE = 2;
 const AUTHORIZE_PAGE = 3;
+const RESET_PAGE = 4;
+const CREATE_OR_RECOVER_PAGE = 5;
+const RECOVER_PAGE = 6;
+const EXPORT_RECOVERY_PHRASE_PAGE = 7;
```

Comment:
This is my fault, but the way we are handling pages like this is a bit of a mess, and we might want to refactor it now that things are getting a little more complex. I would look into using React router. You can check out `manta-front-end` for an example. I _think_ this should work just fine with Tauri, but might not since we aren't in a normal browser environment. If not don't worry about it.